src/pages/studyView/StudyViewUtils.spec.tsx

```diff
@@ -2653,6 +2656,81 @@ describe('StudyViewUtils', () => {
         });
     });
 
+    describe('mergeClinicalDataCollection', () => {
+        it.each([
+            [
+                ({
+                    sampleClinicalData: [
+                        {
+                            sampleId: 's1',
+                            patientId: 'p1',
+                            studyId: 'study1',
+                            uniqueSampleKey: 's1key',
+                            uniquePatientKey: 'p1key',
+                            clinicalAttributeId: 'attr1',
+                            value: 'value1',
+                        },
+                    ],
+                    patientClinicalData: [
+                        {
+                            sampleId: undefined,
+                            patientId: 'p1',
+                            studyId: 'study1',
+                            uniqueSampleKey: undefined,
+                            uniquePatientKey: 'p1key',
+                            clinicalAttributeId: 'attr2',
+                            value: 'value2',
+                        },
+                    ],
+                } as unknown) as ClinicalDataCollection,
+                {
+                    s1key: {
```

Comment:
this seems odd to me.  we are takint a patientClinicalData which has no association with a sample and associating with it with a sample?

Reply:
This is not odd. In the clinical table in Study View each row represents a sample. For each sample row the patient data is appended. So, in the logic to create the row data for the clinical table there is association of patient data.